src/coreclr/jit/fginline.cpp

```diff
@@ -735,6 +735,27 @@ class SubstitutePlaceholdersAndDevirtualizeWalker : public GenTreeVisitor<Substi
             GenTree*    condTree = tree->AsOp()->gtOp1;
             assert(tree == block->lastStmt()->GetRootNode());
 
+            if (condTree->OperIs(GT_COMMA))
```

Comment:
Is there any chance we need to handle `COMMA(COMMA(..., op2))` or the like? I know a few places we have comma handling, we end up having to loop through until the last comma.